main.go

```diff
@@ -54,6 +54,8 @@ const (
 	sopEndpointEnvVarName       = "SOP_ENDPOINT"
 	alertSMTPFromAddrEnvVarName = "ALERT_SMTP_FROM_ADDR"
 	deploymentTypeEnvVarName    = "DEPLOYMENT_TYPE"
+	rhobsApirUrl                = "RHOBS_API_URL"
+	rhssoTokenUrl               = "RH_SSO_TOKEN_URL"
```

Comment:
Please follow naming convention
```suggestion
	rhobsEndpointEnvVarName              = "RHOBS_ENDPOINT"
	rhssoTokenEndpointEnvVarName               = "RHSSO_TOKEN_ENDPOINT"
```

Reply:
Done